src/hotspot/cpu/x86/interp_masm_x86.cpp

```diff
@@ -300,7 +300,7 @@ void InterpreterMacroAssembler::call_VM_base(Register oop_result,
     cmpptr(Address(rbp, frame::interpreter_frame_last_sp_offset * wordSize), NULL_WORD);
     jcc(Assembler::equal, L);
     stop("InterpreterMacroAssembler::call_VM_base:"
-         " last_sp != NULL");
+         " last_sp != nullptr");
```

Comment:
`null`

Reply:
These are textual code fragments so I think `nullptr` is more appropriate - as per changes made to cpu/arm code.

Reply:
Right.